webpack/components/QuestionItem.js

```diff
@@ -69,6 +76,16 @@ class QuestionItem extends Component {
     this.setState(newState);
   }
 
+  updateProgramVar(e){
+    this.setState({programVar : e.target.value});
+  }
+
+  submitProgramVar(e){
+    e.preventDefault();
+    this.hideProgramVarModal();
+    this.props.handleProgramVarChange(this.state.programVar); 
```

Comment:
CDC/SDP-Vocabulary-Service/webpack/components/QuestionItem.js
  86:62  error  Trailing spaces not allowed  no-trailing-spaces